File: packages/insomnia-httpsnippet/test/fixtures/output/go/native/cookies.go
package main

import (
	"fmt"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "http://mockbin.com/har"

	req, _ := http.NewRequest("POST", url, nil)

	req.Header.Add("cookie", "foo=bar; bar=baz")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}

File: packages/insomnia-httpsnippet/test/fixtures/output/node/unirest/cookies.js
var unirest = require('unirest');

var req = unirest('POST', 'http://mockbin.com/har');

var CookieJar = unirest.jar();
CookieJar.add('foo=bar', 'http://mockbin.com/har');
CookieJar.add('bar=baz', 'http://mockbin.com/har');
req.jar(CookieJar);

req.end(function(res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
